src/pages/services.js

```diff
@@ -325,6 +326,48 @@ const ServicesPage = ({ data }) => {
           </div>
         </div>
       </section>
+      <section className="services__ellipses-section">
+        <div className="inner">
+          <h2>Open Standards. Inclusive Practices. Better Outcomes.</h2>
+          <div className="body-large">
+            No matter what problem we’re solving, our core practices and
+            communities power our work and align with current standards for
+            government digital services.
+          </div>
+          <div className="ellipses">
+            <span className="ellipse">
+              <h6>CUSTOMER EXPERIENCE</h6>
```

Comment:
Same feedback here: Write HTML in sentence-case and change to uppercase with CSS.